.cspell.json

```diff
@@ -0,0 +1,23 @@
+// cSpell Settings
+{
+  // Version of the setting file.  Always 0.1
+  "version": "0.1",
+  // language - current active spelling language
+  "language": "en",
+  // words - list of words to be always considered correct
+  "words": [
+    "camelcase",
+    "creds",
+    "kubernetes",
+    "openid",
+    "otomi",
+    "redkubes",
+    "robotv"
+  ],
+  // flagWords - list of words to be always considered incorrect
+  // This is useful for offensive words and common spelling errors.
+  // For example "hte" should be "the"
```

Comment:
I don't think comments are valid `json`

Reply:
but it is working fine. Should I still remove them?